hw/ip/aes/data/aes.hjson

```diff
@@ -154,12 +194,18 @@
         '''
       }
       { bits: "2",
+        name: "IV_CLEAR",
+        desc:  '''
+          Keep current values in IV registers (0) or clear those registers (1).
```

Comment:
Clear to zero, or clear to random value? I think clear to zero is OK from a hamming
distance perspective, there shouldn't be any secrets here. But adding in @tjaychen 
and @cdgori for their thoughts.

Reply:
Currently, everything is cleared to 0. I have clarified this in the hjson.

I think also the IV should be cleared to a random value. Implementing this for all registers is on the top of my todo list.

Reply:
I think it should be random value, but flag it as another thing that is going to be converted to clear-to-random in one big PR (later) and clear to zero for now.